lib/ElementTemplateChooser.css

```diff
@@ -72,6 +73,8 @@
   margin: 0;
   padding: 0;
   list-style: none;
+  overflow-y: scroll;
+  height: calc(65vh - 1.5* var(--text-line-height));
```

Comment:
The result box still looks very large, which is because the height is still fixed.

![image](https://user-images.githubusercontent.com/9433996/158956627-3a6ec065-8eee-4134-b929-063171a06724.png)

Also, the scrollbar should only appear when necessary, e.g. after 4 entries. So something like

```css
overflow-y: auto;
max-height: 300px;
```